packages/webapp/src/contexts/HoneylemonContext.tsx

```diff
@@ -217,15 +217,13 @@ const HoneylemonProvider = ({ children }: HoneylemonProviderProps) => {
   const getPorfolio = async () => {
     setIsPortfolioRefreshing(true);
     const openOrdersRes = await honeylemonService.getOpenOrders(address);
-    console.log(openOrdersRes);
```

Comment:
only nit would be to not leave console logs but it's all good for now because I'm sure there are a bunch elsewhere.

Reply:
does the `-` not mean the line has been removed?